test/extended/router/gatewayapicontroller.go

```diff
@@ -246,6 +249,90 @@ var _ = g.Describe("[sig-network-edge][OCPFeatureGate:GatewayAPIController][Feat
 		g.By("Validating the http connectivity to the backend application")
 		assertHttpRouteConnection(defaultRoutename)
 	})
+
+	g.It("Ensure OSSM subscription, istiod deployment and the istio could be deleted and then get recreated [Serial]", func() {
+		const (
+			operatorNamespace      = "openshift-operators"
+			ingressNamespace       = "openshift-ingress"
+			ossmSubscriptionName   = "servicemeshoperator3"
+			csvName                = "servicemeshoperator3.v3.0.0"
+			ossmOperatorDeployment = "servicemesh-operator3"
+			istiodDeployment       = "istiod-openshift-gateway"
+			istioName              = "openshift-gateway"
+		)
+
+		// deleted the OSSM subscription and then checked if it was restored
+		g.By(fmt.Sprintf("Try to delete the subscription %s", ossmSubscriptionName))
+		_, err := oc.AsAdmin().WithoutNamespace().Run("delete").Args("-n", operatorNamespace, "subscription/"+ossmSubscriptionName).Output()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		g.By(fmt.Sprintf("Wait until the the OSSM subscription %s is automatically created successfully", ossmSubscriptionName))
+		pollWaitSubscriptionCreated(oc, operatorNamespace, ossmSubscriptionName)
```

Comment:
I think this constant is worth a rename cause it sounds too much like the ingress operator's namespace:
```suggestion
		pollWaitSubscriptionCreated(oc, openshiftOperatorsNamespace, ossmSubscriptionName)
```

Reply:
Sure, I will update it.